.github/workflows/check-fixup-commits.yml

```
@@ -0,0 +1,38 @@
+name: Check for fixup commits
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - converted_to_draft
+      - ready_for_review
+      - labeled
+      - unlabeled
+      - auto_merge_enabled
+      - auto_merge_disabled
+
+jobs:
+  check-fixup-commits:
```

Let's name this job/check after what it checks

```suggestion
  no-fixup-commits:
```